app/controllers/users_controller.rb

```diff
@@ -1,5 +1,6 @@
 class UsersController < ApplicationController
-  before_action :correct_user, only: [:show, :edit, :update, :destroy]
+  before_action :correct_user, only: %i[edit update destroy]
+  before_action :set_user, only: %i[show edit update destroy]
```

Comment:
This is backwards right? Also I think show should be protected

Reply:
I wanted to allow users to view other users profiles

Reply:
sgtm